modules/openapi-generator/src/main/resources/cpp-pistache-server/model-source.mustache

```diff
@@ -54,7 +56,21 @@ nlohmann::json {{classname}}::toJson() const
     }
     {{/required}}{{#required}}val["{{baseName}}"] = {{prefix}}ModelBase::toJson(m_{{name}});
     {{/required}}{{/isPrimitiveType}}{{/isListContainer}}{{/vars}}
-
+    {{#isEnum}}{{#allowableValues}}
```

Comment:
does this not belong in the vars section of the template?

Reply:
Please see my comment [here](https://github.com/OpenAPITools/openapi-generator/pull/1366#discussion_r230570943)